### DISCUSSION TITLE
Working with noSQL dbs in ninja-api

### DISCUSSION THREAD

**Discussion by GhostMEn20034** [vitalik/django-ninja#680] [Q&A] [answered]
Can I work with noSQL dbs in ninja-api as it can be done in FAST API?

**Comment by vitalik** [comment, marked as answer]
@GhostMEn20034 

sure, works like a charm

**Reply by GhostMEn20034** [reply]
Are there examples in docs?

**Reply by GhostMEn20034** [reply]
Because I've not found it

**Reply by vitalik** [reply]
well there is nothing specific to document - just use regular methods to get/list/insert objects

example with pymongo:

```Python

client = MongoClient()
db = client.some_database

@api.get("/list/from/mongo")
def example(request):
    return list(db.somecollection.find())
```

**Reply by GhostMEn20034** [reply]
Дякую за допомогу!
